utils/api/definitions/fields.ts

```diff
@@ -23,11 +23,11 @@ enum FieldType {
 export const fieldDefinitions: FieldDefinitions = {
   [FieldKey.INCOME]: {
     key: FieldKey.INCOME,
-    label: 'What is your current net income?',
+    label: 'What is your current annual net income in Canadian Dollars?',
     category: FieldCategory.INCOME_DETAILS,
     order: 1,
     type: FieldType.NUMBER,
-    placeholder: '20000',
+    placeholder: '$20,000',
```

Comment:
Updated placeholder in API here, just to match spec a little tighter -- doesn't seem to be any test implication